### PR TITLE
Update dependency go modules for k8s v1.26.0

### PR DESCRIPTION
**Pull request by sunnylovestiramisu**
Ran kubernetes-csi/csi-release-tools go-get-kubernetes.sh -p 1.26.0

### CODE REVIEW

File: client/go.mod [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 
-go 1.18
+go 1.19
 
 require (
-	k8s.io/api v0.26.0-rc.0
-	k8s.io/apimachinery v0.26.0-rc.0
-	k8s.io/client-go v0.26.0-rc.0
+	k8s.io/api v0.26.0
+	k8s.io/apimachinery v0.26.1-rc.0
```
Comment (xing-yang) [inline client/go.mod:7]: Why is this v0.26.1-rc.0, not v0.26.0?

File: client/go.mod [hunk 2]
```diff
@@ -31,11 +31,11 @@ require (
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
 	golang.org/x/mod v0.6.0 // indirect
-	golang.org/x/net v0.1.1-0.20221027164007-c63010009c80 // indirect
+	golang.org/x/net v0.3.1-0.20221206200815-1e63c2f08a10 // indirect
```
Comment (xing-yang) [inline client/go.mod:34]: This one is causing the trivy scan failure. Is this golang.org/x/net v0.3.1 referenced in other sidecar's go.mod?
Can you check who is using this lib?
One way to workaround this problem is to manually add a replace statement:
replace (
  golang.org/x/net => v0.4.0
)

File: client/go.mod [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
+go 1.19
 
 require (
-	k8s.io/api v0.26.0-rc.0
-	k8s.io/apimachinery v0.26.0-rc.0
-	k8s.io/client-go v0.26.0-rc.0
+	k8s.io/api v0.26.0
+	k8s.io/apimachinery v0.26.0
+	k8s.io/client-go v0.26.0
 	k8s.io/code-generator v0.25.2
```
Comment (sunnylovestiramisu) [inline client/go.mod:9]: Does this one need to be 0.26.0 as well?

File: go.mod [hunk 4]
```diff
@@ -17,11 +17,11 @@ require (
...
-	k8s.io/client-go v0.26.0-rc.0
-	k8s.io/component-base v0.26.0-rc.0
-	k8s.io/component-helpers v0.26.0-rc.0
+	k8s.io/api v0.26.0
+	k8s.io/apimachinery v0.26.1-rc.0
+	k8s.io/client-go v0.26.0
+	k8s.io/component-base v0.26.0
+	k8s.io/component-helpers v0.26.0
 	k8s.io/klog/v2 v2.80.1
 	k8s.io/kubernetes v1.25.4
```
[the remark this answers is not in the document]
Comment (sunnylovestiramisu) [reply go.mod:26]: And because it is depending on k8s should we keep the replace generated by the script?

File: client/go.mod (hunk 3 above)
Comment (xing-yang) [reply client/go.mod:9]: This one is okay. Leave this one as is.

File: go.mod (hunk 4 above)
[the remark this answers is not in the document]
Comment (xing-yang) [reply go.mod:26]: Yes, this should be v1.26.0 as well.  I don't think we need the replace as it points to the same version any way.

File: client/go.mod (hunk 2 above)
Comment (sunnylovestiramisu) [reply client/go.mod:34]: Let me try to update this

File: go.mod [hunk 5]
```diff
@@ -17,13 +17,13 @@ require (
 	github.com/prometheus/common v0.37.0
 	github.com/spf13/cobra v1.6.1
 	google.golang.org/grpc v1.50.1
-	k8s.io/api v0.26.0-rc.0
-	k8s.io/apimachinery v0.26.0-rc.0
-	k8s.io/client-go v0.26.0-rc.0
-	k8s.io/component-base v0.26.0-rc.0
-	k8s.io/component-helpers v0.26.0-rc.0
+	k8s.io/api v0.26.0
+	k8s.io/apimachinery v0.26.1-rc.0
```
Comment (xing-yang) [inline go.mod:21]: Any reason why this one is v0.26.1-rc.0, not v0.26.0?

Comment (sunnylovestiramisu) [reply go.mod:21]: Manually updated it, somehow the go-get-k8s.sh picked this one up

File: client/go.mod (hunk 2 above)
Comment (xing-yang) [inline client/go.mod:34]: @sunnylovestiramisu Can you submit another PR to make a similar change in client/go.mod as follows:
replace golang.org/x/net => golang.org/x/net v0.4.0

I'm getting a dependencies change error when generating the changelog.  I wonder if it is because the client and server go.mod are having different libs.
